tests/test_crew_ai/test_agents.py

```
@@ -0,0 +1,64 @@
+  - Returns empty string when no context available
+  - Unknown agent type returns empty string
+"""
+
+import pytest
+from unittest.mock import patch, MagicMock
+
+
+class TestGetAgentContext:
+    """Tests for RobotAgents._get_agent_context."""
+
+    def _make_agents(self, optimization_enabled=False, library_context=None):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_make_agents`

(ANN202)